.travis.yml

```diff
@@ -14,3 +15,12 @@ cache:
 script:
  - cd whitepaper
  - make latex
+ - make github
```

Comment:
Do not merge yet, I need to find a way to execute this command only on  master branch